packages/protocol/contracts/L1/v1/V1Utils.sol

```diff
@@ -98,11 +98,18 @@ library V1Utils {
         LibData.State storage state,
         LibData.ForkChoice storage fc,
         uint256 blockId
+    ) internal view returns (uint64) {
+        return fc.provenAt + getDelay(state, blockId);
+    }
+
+    function getDelay(
```

Comment:
Should we rename it to `getUncleProofDelay` and also rename `uncleProofDeadline` to `getUncleProofDeadline`